src/style.css

```
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
```

```suggestion
  background: url(images/star.svg) no-repeat center;
  height: 16px;
  width: 16px;
```

can be a bit simplified